group.go

```diff
@@ -52,87 +55,111 @@ func (r *Group) Middleware(middlewares ...httpcontract.Middleware) route.Route {
 	r.clearMiddlewares()
 }
 
 func (r *Group) Resource(relativePath string, controller httpcontract.ResourceController) {
-	r.getRoutesWithMiddlewares().Get(pathToFiberPath(relativePath), []fiber.Handler{handlerToFiberHandler(controller.Index)}...)
-	r.getRoutesWithMiddlewares().Post(pathToFiberPath(relativePath), []fiber.Handler{handlerToFiberHandler(controller.Store)}...)
-	r.getRoutesWithMiddlewares().Get(pathToFiberPath(relativePath+"/{id}"), []fiber.Handler{handlerToFiberHandler(controller.Show)}...)
-	r.getRoutesWithMiddlewares().Put(pathToFiberPath(relativePath+"/{id}"), []fiber.Handler{handlerToFiberHandler(controller.Update)}...)
-	r.getRoutesWithMiddlewares().Patch(pathToFiberPath(relativePath+"/{id}"), []fiber.Handler{handlerToFiberHandler(controller.Update)}...)
-	r.getRoutesWithMiddlewares().Delete(pathToFiberPath(relativePath+"/{id}"), []fiber.Handler{handlerToFiberHandler(controller.Destroy)}...)
+	r.getRoutesWithMiddlewares(relativePath).Get(pathToFiberPath(relativePath), []fiber.Handler{handlerToFiberHandler(controller.Index)}...)
+	r.getRoutesWithMiddlewares(relativePath).Post(pathToFiberPath(relativePath), []fiber.Handler{handlerToFiberHandler(controller.Store)}...)
+	r.getRoutesWithMiddlewares(relativePath).Get(pathToFiberPath(relativePath+"/{id}"), []fiber.Handler{handlerToFiberHandler(controller.Show)}...)
+	r.getRoutesWithMiddlewares(relativePath).Put(pathToFiberPath(relativePath+"/{id}"), []fiber.Handler{handlerToFiberHandler(controller.Update)}...)
+	r.getRoutesWithMiddlewares(relativePath).Patch(pathToFiberPath(relativePath+"/{id}"), []fiber.Handler{handlerToFiberHandler(controller.Update)}...)
+	r.getRoutesWithMiddlewares(relativePath).Delete(pathToFiberPath(relativePath+"/{id}"), []fiber.Handler{handlerToFiberHandler(controller.Destroy)}...)
 	r.clearMiddlewares()
 }
 
 func (r *Group) Static(relativePath, root string) {
-	r.getRoutesWithMiddlewares().Static(pathToFiberPath(relativePath), root)
+	r.getRoutesWithMiddlewares(relativePath).Static(pathToFiberPath(relativePath), root)
 	r.clearMiddlewares()
 }
 
 func (r *Group) StaticFile(relativePath, filepath string) {
-	r.getRoutesWithMiddlewares().Static(pathToFiberPath(relativePath), filepath)
+	r.getRoutesWithMiddlewares(relativePath).Static(pathToFiberPath(relativePath), filepath)
 	r.clearMiddlewares()
 }
 
 func (r *Group) StaticFS(relativePath string, fs http.FileSystem) {
-	r.getRoutesWithMiddlewares().Use(pathToFiberPath(relativePath), filesystem.New(filesystem.Config{
+	r.getRoutesWithMiddlewares(relativePath).Use(pathToFiberPath(relativePath), filesystem.New(filesystem.Config{
 		Root: fs,
 	}))
 	r.clearMiddlewares()
 }
 
-func (r *Group) getRoutesWithMiddlewares() fiber.Router {
+func (r *Group) getRoutesWithMiddlewares(relativePath string) fiber.Router {
 	prefix := pathToFiberPath(r.originPrefix + "/" + r.prefix)
+	fullPath := pathToFiberPath(prefix + "/" + relativePath)
+
 	r.prefix = ""
 	fiberGroup := r.instance.Group(prefix)
 
 	var middlewares []any
```

Comment:
It's better to use `[]fiber.Handler` here.

Reply:
> It's better to use `[]fiber.Handler` here.

Can't do it, because fiber's `Use` method only receive any type.